industrydive/django-site-metatags
# TECH-6903: Run modernize on django-site-metatags

[TECH-6903: Run modernize on django-site-metatags](https://industrydive.atlassian.net/browse/TECH-6903)

### Non-technical Context
N/A

### Technical Context
Ran modernize on the repo and made changes

### Reproduction Steps
N/A

### Manual Testing Instructions
- Run `python-modernize`
- There should be no unapplied suggestions
- Do the testing for the divesite PR [here](https://github.com/industrydive/divesite/pull/5063)
### QA Checklist

#### Developer

- [x] Verify modernize output does not contain any results for **new** Python files
- [x] Set deployment notes on Jira ticket


#### Reviewer

    - If there are few or minor issues, add inline GitHub comments
    - If there are many issues or it seems like the developer did not follow the style guide, leave a comment asking them to do so
- [x] Verify modernize output does not contain any results for **new** Python files
- [x] Read any deployment notes on the Jira ticket and ensure they can be followed
